x-pack/filebeat/module/threatintel/misp/ingest/pipeline.yml

```diff
@@ -288,6 +317,23 @@ processors:
         - threatintel.misp.attribute.value
       ignore_missing: true
       if: ctx?.threatintel?.indicator?.type != 'unknown'
+  - remove:
+      field:
+        - threatintel.misp.object.template_version
+        - threatintel.misp.object.template_uuid
+        - threatintel.misp.object.description
+        - threatintel.misp.object.distribution
```

Comment:
Is this removing all the data under `threatintel.misp.object.` ? I couldn't see any in the expected.json.

If we are removing all, you can simply remove `threatintel.misp.object` instead of having to define all the fields manually.

Reply:
I partly spelled all these out to raise them here to make sure they are actually things that we want to drop. If they are, I'll squash to `threatintel.misp.object`.

Reply:
Done